Src/WitsmlExplorer.Frontend/components/Sidebar/ServerManager.tsx

```diff
@@ -121,79 +134,126 @@ const ServerManager = (): React.ReactElement => {
-            </ListItemIcon>
-            <Typography color={"initial"}>Add server</Typography>
-          </MenuItem>
-        </Select>
-
-        <AuthenticationState />
-      </FormControl>
+      <Header>
+        <Typography color={"primary"} bold={true}>
+          Manage Connections
+        </Typography>
+        <Button variant="outlined" value={NEW_SERVER_ID} key={NEW_SERVER_ID} onClick={() => onEditItem(emptyServer())}>
+          <Icon name="cloudDownload" />
+          New server
+        </Button>
+      </Header>
+      <EdsProvider density={theme}>
+        <Table style={{ width: "100%" }} className="serversList">
+          <Table.Head>
+            <Table.Row>
+              <Table.Cell style={CellHeaderStyle}>Server Name</Table.Cell>
+              <Table.Cell style={CellHeaderStyle}>Server</Table.Cell>
+              <Table.Cell style={CellHeader}>Test Connection</Table.Cell>
+              <Table.Cell style={CellHeaderStyle}>Status</Table.Cell>
+              <Table.Cell></Table.Cell>
+              <Table.Cell></Table.Cell>
+            </Table.Row>
+          </Table.Head>
+          <Table.Body>
+            {servers
+              .sort((a, b) => a.name.localeCompare(b.name))
+              .map((server: Server) => (
+                <Table.Row id={server.id} key={server.id}>
+                  <Table.Cell style={CellHeaderStyle}>{server.name}</Table.Cell>
+                  <Table.Cell style={CellHeaderStyle}>{server.url}</Table.Cell>
+                  <Table.Cell style={{ textAlign: "center" }}>
+                    <Icon color={selectedServer?.id == server.id ? colors.interactive.successResting : colors.text.staticIconsTertiary} name="cloudDownload" />
+                  </Table.Cell>
+                  <Table.Cell style={CellHeaderStyle}>
+                    <EdsProvider density={"compact"}>
+                      <CustomButton
+                        variant="outlined"
+                        onClick={() => onSelectItem(server)}
+                        style={{
+                          color: selectedServer?.id == server.id && wells.length ? colors.interactive.primaryResting : colors.text.staticIconsDefault,
+                          borderColor: selectedServer?.id == server.id && wells.length ? colors.interactive.successResting : colors.text.staticIconsDefault
+                        }}
+                      >
+                        {selectedServer?.id == server.id && wells.length ? "Connected" : "Connect"}
+                      </CustomButton>
+                    </EdsProvider>
+                  </Table.Cell>
+                  <Table.Cell style={CellHeaderStyle}>
+                    <Icon name="edit" size={24} onClick={() => onEditItem(server)} />
+                  </Table.Cell>
+                  <Table.Cell style={CellHeaderStyle}>
+                    <Icon name="deleteToTrash" size={24} onClick={() => showDeleteModal(server)} />
```

Comment:
Icons with onClick lack the properties of a button such as highlighting on hover and changing the cursor to a pointer (it seems that we were missing these properties on the icons in the ServerManager from before). Have you tried the ghost_icon variant of the EDS button to improve this? https://eds-storybook-react.azurewebsites.net/?path=/docs/inputs-button-button--icon-button

Reply:
updated!